cohort_4/playbook/ACTIVITIES.md

```diff
@@ -756,3 +760,78 @@ the technical mentors for Hack Camp will be pinged. For example:
 If you'd like to be added/removed from the list, let
 [@zachlatta](https://github.com/zachlatta) know and he'll get right on it (say
 `slackbot help` somewhere in the chat to see who's currently on the list).
+
+# Communicating with the Group
+
+In this modern day and age, we're fortunate to have _not one_, _**not two**_,
+**but THREE** methods of communication in camp.
+
+1. Announcing to the whole group
+2. Sharing an announcement to each hacker individually
+3. Using `@channel` on Slack
+
+_But when do we use these marvelous methods of communication, you ask?_
+
+## Announcing to the Whole Group
```

Comment:
Brought up as in one of them saying it’d be valuable?

On Wednesday, August 5, 2015, Max Wofford <notifications@github.com> wrote:

> In cohort_4/playbook/ACTIVITIES.md
> <https://github.com/hackedu/hack-camp/pull/339#discussion_r36371711>:
>
> > @@ -756,3 +760,78 @@ the technical mentors for Hack Camp will be pinged. For example:
> >  If you'd like to be added/removed from the list, let
> >  [@zachlatta](https://github.com/zachlatta) know and he'll get right on it (say
> >  `slackbot help` somewhere in the chat to see who's currently on the list).
> > +
> > +# Communicating with the Group
> > +
> > +In this modern day and age, we're fortunate to have _not one_, _**not two**_,
> > +**but THREE** methods of communication in camp.
> > +
> > +1. Announcing to the whole group
> > +2. Sharing an announcement to each hacker individually
> > +3. Using `@channel` on Slack
> > +
> > +_But when do we use these marvelous methods of communication, you ask?_
> > +
> > +## Announcing to the Whole Group
>
> Perhaps if it brought up by a club leader.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hack-camp/pull/339/files#r36371711>.
>


-- 
--
Zach Latta
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta